src/Export/Field/FilterAttributes.php

```diff
@@ -26,8 +31,30 @@ public function getValue(Product $product): string
     {
         $attributes = $product->getChildren()->reduce(function (array $result, Product $child): array {
             return $result + array_map($this->propertyFormatter, $child->getOptions()->getElements());
-        }, array_map($this->propertyFormatter, $product->getProperties()->getElements()));
+        }, array_map($this->propertyFormatter, $this->applyPropertyGroupsFilter($product)));
 
         return $attributes ? '|' . implode('|', array_values($attributes)) . '|' : '';
     }
+
+    private function applyPropertyGroupsFilter(Product $product): array
+    {
+        $disabledPropertyGroups =  $this->excludedFields->getDisabledPropertyGroups();
+        $productProperties = $product->getProperties()->getElements();
+
+        if ($disabledPropertyGroups) {
```

Comment:
could we simplify this block to `  return $product->getProperties()->filter(function(PropertyGroupOptionEntity $option) use ($disabledPropertyGroups) {
            return !in_array($option->getGroupId(), $disabledPropertyGroups);
        })->getElements();`
?